.github/workflows/trivy.yml

```diff
@@ -28,24 +28,20 @@ jobs:
           repository_url: "ghcr.io/spacelift-io/vcs-agent"
 
       - name: Run Trivy vulnerability scanner (amd64)
-        uses: aquasecurity/trivy-action@0.29.0
+        uses: aquasecurity/trivy-action@master
```

Comment:
so that we don't need to maintain it in the future

Reply:
Can we use latest Trivy action versions, but pin the Trivy version?

IIIRC I experienced a lot of breaking changes and changes in trivy when replacing tfsec on the infra repo, that could be nice if we could avoid this pain here.

Trivy is anyway pulling the policy bundle every times it runs, so we'll benefit of the latest security rules without having to bump.anything here.